cfpb/cfgov-refresh
# Adding Fixes for Date Range on blog

Removed date formatting code from the client. This is now handled in Sheer. The clients responsibility is just validation.

## Additions

- Added shared object for `filterable-list-control.js` acceptance test.

## Removals

- Removed code deal with date formatting code from the client.

## Changes

- Modified Sheer code to use `date parser` when date format is `mm/dd` 

## Testing

- Visit `/blog` and enter `05/11/2015` in the from date. Click `Apply Filters` and you should see results.
- Enter '05/11' in the from date. Click `Apply Filters` and you should see results.
- Enter '04/fff' in the from date. Click `Apply Filters` and you should see an error notification.

## Review

- @anselmbradford 
- @jimmynotjim 
- @KimberlyMunoz 
- @rosskarchner 
- @kurtw 

## Screenshots

- 
<img width="826" alt="screen shot 2016-03-28 at 10 44 23 am" src="https://cloud.githubusercontent.com/assets/1696212/14080358/87305d50-f4d2-11e5-8e75-8ccb4448c630.png">

- 
<img width="836" alt="screen shot 2016-03-28 at 10 48 30 am" src="https://cloud.githubusercontent.com/assets/1696212/14080373/a2572ad2-f4d2-11e5-9571-95ab51d3f47a.png">




## Notes

- There is currently an issue where the Filters aren't remaining expanded when there are active filters on Sheer pages. I will address that issue in a separate PR.

## Todos

- Add more acceptance tests.

## Checklist
